src/main/java/Duke.java

```diff
@@ -1,10 +1,202 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
```

Comment:
Import statements are not following java coding standard.(should be in lexographic order)